build/lib/release.sh

```diff
@@ -282,17 +298,20 @@ function kube::release::build_conformance_image() {
 # This builds all the release docker images (One docker image per binary)
 # Args:
 #  $1 - binary_dir, the directory to save the tared images to.
-#  $2 - arch, architecture for which we are building docker images.
+#  $2 - os, the OS for which we are building docker images.
```

Comment:
should OS be added to the end of this so it can be optional and not break existing tooling?